test/org/zaproxy/zap/extension/dynssl/SslCertificateUtilsUnitTest.java

```diff
@@ -0,0 +1,153 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ * 
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ * 
+ * Copyright 2016 The ZAP Development Team
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.extension.dynssl;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.junit.Assert.assertThat;
+
+import java.nio.charset.StandardCharsets;
+
+import org.junit.Test;
+
+/**
+ * Unit test for {@link SslCertificateUtils}.
+ */
+public class SslCertificateUtilsUnitTest {
+
+    private static final String CERT_DATA = "Certificate data...";
+    private static final String CERT_DATA_BASE64 = "Q2VydGlmaWNhdGUgZGF0YS4uLg==";
```

Comment:
Would it be better to b64 encode during execution? That way if the value of `CERT_DATA` is change then the encoded version doesn't have to be updated? (Same for the key below.)